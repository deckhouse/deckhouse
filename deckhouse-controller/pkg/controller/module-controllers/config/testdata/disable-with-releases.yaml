---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleConfig
metadata:
  name: test-module
spec:
  enabled: false
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  name: test-module
properties:
  source: test-source
status:
  phase: Ready
  conditions:
    - type: EnabledByModuleConfig
      status: "True"
---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleRelease
metadata:
  name: test-module-v1.0.0
  labels:
    module: test-module
spec:
  moduleName: test-module
  version: 1.0.0
status:
  phase: Pending 