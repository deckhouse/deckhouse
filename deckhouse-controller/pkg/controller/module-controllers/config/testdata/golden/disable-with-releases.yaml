---
apiVersion: deckhouse.io/v1alpha1
kind: ModuleConfig
metadata:
  creationTimestamp: null
  name: test-module
  resourceVersion: "1000"
spec:
  enabled: false
status:
  message: ""
  version: "1"
---
apiVersion: deckhouse.io/v1alpha1
kind: Module
metadata:
  creationTimestamp: null
  name: test-module
  resourceVersion: "1000"
properties:
  source: test-source
status:
  conditions:
  - lastProbeTime: null
    lastTransitionTime: null
    status: "False"
    type: EnabledByModuleConfig
  - lastProbeTime: null
    lastTransitionTime: null
    message: disabled
    reason: Disabled
    status: "False"
    type: IsReady
  - lastProbeTime: null
    lastTransitionTime: null
    status: Unknown
    type: LastReleaseDeployed
  phase: Ready
